_people/Marlo_Morales.markdown

```diff
@@ -0,0 +1,12 @@
+---
+name: Marlo Morales
+lastname: Morales
+group: alumni
```

Comment:
@MarloMo should this be `group: students`?

Reply:
I graduated last year @duetosymmetry

Reply:
Oops, congrats!

Reply:
Thank you!